pkg/apis/apis/v1alpha1/types_apiexport.go

```diff
@@ -209,6 +212,22 @@ type PermissionClaim struct {
 	IdentityHash string `json:"identityHash,omitempty"`
 }
 
+type Verbs struct {
+	// claimed is a list of verbs that restrict access via services.
+	// '*' represents all verbs.
+	//
+	// +required
+	Claimed []string `json:"claimed"`
+
+	// restrictTo is a list of verbs that restrict access via workspaces.
+	// They also restrict access via services if the same resources
+	// are claimed by a different API export in the same workspace.
+	// '*' represents all verbs.
+	//
+	// +required
+	RestrictTo []string `json:"restrictTo"`
```

Comment:
so the constaint of no-overlap is not implemented?

Reply:
🔴  We cannot add that later without a breaking change

Reply:
ack, let me implement the overlap then here 👍 I was planning to do it as a follow-up, but agree, this cannot be added later without a breaking change, agreed.